modules/andes-core/broker/src/main/java/org/wso2/andes/subscription/SubscriptionStore.java

```diff
@@ -609,6 +672,50 @@ public synchronized void createDisconnectOrRemoveLocalSubscription(LocalSubscrip
             channelIdMap.remove(subscription.getChannelID());
         }
 
+
+        List<AndesQueue> allQueuesStored = andesContextStore.getAllQueuesStored();
+        for (AndesQueue andesQueue : allQueuesStored) {
+            if (subscription.isDurable() && !subscription.isBoundToTopic() && andesQueue.isExclusiveConsumer && andesQueue.queueName.equalsIgnoreCase(subscription.getTargetQueue())) {
```

Comment:
This if condition checks lots of conditions and it reduce the readability. Can simplify this?